### ISSUE TITLE
DT_Coordinate incompatibility

### ISSUE THREAD

**Issue by mike9k1** [emoose/DQXIAsset#4]
![image](https://user-images.githubusercontent.com/31750577/122646613-a177c000-d0e5-11eb-99d9-42ac52176f2e.png)

**Comment by emoose** [comment]
Yeah sadly that's one of the main DTs I couldn't figure out, if you take a look at the struct for it in the SDK it's pretty much just a bunch of fields all of the same type, and then in the DT file itself they've been shifted around & re-ordered or something...

That reordering happened to a bunch of DT's going from SDK -> UAsset, but since fields usually had different types it wasn't too difficult to figure out what each one was, with DT_Coordinate all using the same type for everything that made it pretty hard for me though :/

If anyone knows more about how that coordinate stuff works maybe it could be figured out though, comparing other structs from SDK -> DQXIAsset should show how to add things to here, it's pretty similar really.

**Comment by mike9k1** [comment]
Okay -- thank you emoose :)

I suspected I wasn't the only one to run across this, but wasn't sure if it had been reported yet. I may do some poking around with it at some point. It is believed this is the DataTable that is referenced for coordinating character meshes, materials, etc.

On the plus side: other coordinate DataTables that i've tested seem to work with DQXIAsset fine.

**Comment by mike9k1** [comment]
I know this is suuuper old, and it took me a long while to come back to this one, but I figured i'd add some notes here. If nothing else in the off chance it helps someone else out or makes an update / separate tool for parsing DT_Coordinate possible:

We figured out a little bit more of how DT_Coordinate works, mainly by just finding the header indexes and matching them with data (as int32 little endian hex) in the .uexp. I'll add a quick excerpt below:

Row starting at 0x170cb1:

"Coord_P001_E001,0x00,0x0188 (not a header value), /Game/Characters/Human/P001/Base/Mesh/SK_P001.SK_P001, /Game/Characters/Human/P001/Wear/Mesh/SK_P001_Wear_E001.SK_P001_Wear_E001, None, None, None, None, None, /Game/Characters/Human/P001/Face/Mesh/SK_P001_Face001_E001.SK_P001_Face001_E001, None, None, None, None, None, /Game/Characters/Human/P001/Hair/Mesh/SK_P001_Hair001_E001.SK_P001_Hair001_E001, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None"
